jscomp/others/js_json.ml

```diff
@@ -52,6 +52,31 @@ let reifyType (type a) (x : 'a) : (a kind * a ) =
 
 let reify_type = reifyType 
 
+type tagged_t = 
```

Comment:
The `t` could be implicit, so just `tagged` perhaps? Alternatively, I've seen reason code using the form `taggedT`. Or the very explicit `taggedJson`.

(Personally I think I'd prefer snake case for type names in general to distinguish them from let bindings, but that doesn't seem to be the winning position)